src/components/markdown-htmlview.component.js

```diff
@@ -0,0 +1,356 @@
+import React, { Component } from 'react';
+import { Dimensions, StyleSheet, View, Text, Platform } from 'react-native';
+import HTMLView from 'react-native-htmlview';
+import marked from 'marked';
+
+import { ImageZoom } from 'components';
+import { colors, emojis, fonts, normalize } from 'config';
+
+const lightFont = {
+  ...fonts.fontPrimaryLight,
+};
+
+const regularFont = {
+  ...fonts.fontPrimary,
+};
+
+const textStyleLight = {
+  fontSize: Platform.OS === 'ios' ? normalize(11) : normalize(12),
+  color: colors.primaryDark,
+  ...lightFont,
```

Comment:
I know this is a really controlled environment but you should write this like:
```js
const textStyleLight = {
  ...lightFont,
  fontSize: Platform.OS === 'ios' ? normalize(11) : normalize(12),
  color: colors.primaryDark
}
```
By doing that you'll avoid potential overwrites in the new object.

Reply:
definitely (wasn't well aware of that, thanks)